File: 1550-Three-Consecutive-Odds/1550.c
bool threeConsecutiveOdds(int* arr, int arrSize) {

    char foundConsec = 0;
    
    for(ushort i=0; i<arrSize; i++) {
        if(arr[i] % 2 == 1) foundConsec++;
        else foundConsec = 0;

        if(foundConsec==3) return true;
    }

    return false;
}

File: 1550-Three-Consecutive-Odds/1550.rs
impl Solution {
    pub fn three_consecutive_odds(arr: Vec<i32>) -> bool {
        let mut foundConsec = 0;

        for n in arr {
            if (n % 2 == 1) {
                foundConsec += 1;

                if (foundConsec == 3) {
                    return true;
                }
            } else {
                foundConsec = 0;
            }
        }

        false
    }
}
